packages/icons/README.md

```diff
@@ -0,0 +1,21 @@
+# @vibe/icons
+
+This package includes all of the icons of monday.com's Vibe Design System
```

Comment:
Let's add a link to the catalog?

Reply:
Worth mentioning that it contains React icons and SVGs

Reply:
I added about the react icons, right now it doesn't export the svgs, when we add the svgs will add here too